### PR TITLE
Update database schema

### PR DESCRIPTION
**Pull request by m4dkip**
Not sure how to translate the Reviews part into schema

### CODE REVIEW

File: src/main/resources/data.sql [hunk 1]
```diff
@@ -1,9 +1,57 @@
...
+	fsid INTEGER PRIMARY KEY,
+	daysOption daysEnum,
+	shiftsOption shiftsEnum
+);
+
+-- fsid = id of assigned schedule
+CREATE TABLE FTRiders (
+	drid INTEGER PRIMARY KEY,
+	fsid INTEGER,
+	FOREIGN KEY (drid) REFERENCES DeliveryRiders,
```
Comment (GabrielYik) [inline src/main/resources/data.sql]: Should this also include `ON DELETE CASCADE`?

File: src/main/resources/data.sql [hunk 2]
```diff
@@ -1,9 +1,57 @@
...
+	startHour hoursEnum,
+	endHour hoursEnum,
+	FOREIGN KEY (psid) REFERENCES PTSchedules ON DELETE CASCADE
+);
+
+-- psid = id of assigned schedule
+CREATE TABLE PTRiders (
+	drid INTEGER PRIMARY KEY,
+	psid INTEGER,
+	FOREIGN KEY (drid) REFERENCES DeliveryRiders,
```
Comment (GabrielYik) [inline src/main/resources/data.sql]: Should this also include `ON DELETE CASCADE`?

File: src/main/resources/data.sql [hunk 3]
```diff
@@ -12,15 +60,72 @@ CREATE TABLE Customers (
...
+
+CREATE TABLE Reviews (
+	rvid INTEGER PRIMARY KEY,
+	rating INTEGER,
+	content VARCHAR(1000)
+);
+
 CREATE TABLE Orders (
     oid INTEGER PRIMARY KEY,
+	drid INTEGER,
```
Comment (GabrielYik) [inline src/main/resources/data.sql]: I think we could have a customer id here too. Can help me add?

File: src/main/resources/data.sql [hunk 4]
```diff
@@ -12,15 +60,72 @@ CREATE TABLE Customers (
...
     restaurantArrivalTime TIMESTAMP,
     restaurantDepartureTime TIMESTAMP,
-    deliveryTime TIMESTAMP
-)
+    deliveryTime TIMESTAMP,
+	FOREIGN KEY (drid) REFERENCES DeliveryRiders,
+	FOREIGN KEY (rvid) REFERENCES Reviews
+);
+
+CREATE TABLE OrdersContains (
```
Comment (GabrielYik) [inline src/main/resources/data.sql]: I think we can do away with `OrderContains` and use the `Reviews` table for this instead

File: src/main/resources/data.sql [hunk 5]
```diff
@@ -12,15 +60,72 @@ CREATE TABLE Customers (
...
+	rid INTEGER,
+	name VARCHAR(100),
+	category VARCHAR(100),
+	price MONEY,
+	dailyLimit INTEGER,
+	FOREIGN KEY (rid) REFERENCES Restaurants ON DELETE CASCADE
+);
+
+CREATE TABLE Reviews (
+	rvid INTEGER PRIMARY KEY,
```
Comment (GabrielYik) [inline src/main/resources/data.sql]: * Add food item id
* Add order id

**Comment by m4dkip** [comment]
Thanks, have made the changes and also change all the tabs into spaces

**Pull request merged**